module/zfs/vdev_indirect.c

```diff
@@ -567,6 +567,12 @@ spa_condense_indirect_commit_entry(spa_t *spa,
 	dmu_tx_commit(tx);
 }
 
+#define	delay_ms(ms) \
+	do { \
+		if (ms) \
+			delay(MSEC_TO_TICK(ms)); \
+	} while (0);
```

Comment:
Let's use `zfs_sleep_until`() rather than introducing a new macro for this.

Reply:
Done.